qemu/tests/cfg/watchdog.cfg

```
@@ -3,6 +3,8 @@
     enable_watchdog = yes
     start_vm = no
     type = watchdog
+    aarch64:
+        watchdog_type_check = " -M virt -watchdog ?"
```

```suggestion
        watchdog_type_check = " -M virt -watchdog '?'"
```
Or use `help` instead.

---

Thank you for your careful review.